docs/tracking.md

```diff
@@ -93,6 +94,8 @@ please open an issue on [this issue tracker](https://github.com/Quansight-Labs/f
 | PyAV                 |        [:simple-github:](https://github.com/PyAV-Org/PyAV/issues/1956)         |                       |                            |                            |                                                            |                                                                                              |
 | pybind11             |       [:simple-github:](https://github.com/pybind/pybind11/issues/5112)        | :material-check-bold: |   :material-check-bold:    |            2.13            |                                                            |                                                                                              |
 | pycares              |        [:simple-github:](https://github.com/saghul/pycares/issues/213)         | :material-check-bold: |                            |                            |                                                            |                                                                                              |
+| pycocotools          |         [:simple-github:](https://github.com/ppwwyyxx/cocoapi/pull/27)         | :material-check-bold: |   :material-check-bold:    |           2.0.10           |                                                            |                                                                                              |
+| pycryptodome         |    [:simple-github:](https://github.com/Legrandin/pycryptodome/issues/891)     | :material-check-bold: |   :material-check-bold:    |           3.23.0           |                                                            |                                                                                              |
```

Comment:
That issue has no info on initial support, it looks like https://github.com/Legrandin/pycryptodome/issues/813 is the correct one. Can you change to that? Also, you may want to cross-link the issue you opened last week to that one.